### PR TITLE
docs(guides) update typescript guide; add missing contribution after …

### LINKED ISSUE #3561: 'Getting Started' guide + Typescript guide default import build error
**What is the current behavior?**
Following the steps in [Getting Started](https://v4.webpack.js.org/guides/getting-started/) everything works fine.
Moving on to the guide for [Typescript](https://v4.webpack.js.org/guides/typescript/), copying index.js into index.ts and following the steps results in a build failure
`ERROR in C:\webpack-demo\src\index.ts
./src/index.ts
[tsl] ERROR in C:\webpack-demo\src\index.ts(1,8)
      TS1259: Module '"C:/webpack-demo/node_modules/@types/lodash/ts3.1/index"' can only be default-imported using the 'allowSyntheticDefaultImports' flag`

**If the current behavior is a bug, please provide the steps to reproduce.**
It's not a bug but the guide implicitly suggests that the following should work
Follow the instructions in [Getting Started](https://v4.webpack.js.org/guides/getting-started/)
`$ cp index.js index.ts`
Follow the instructions in  [Typescript](https://v4.webpack.js.org/guides/typescript/)

**What is the expected behavior?**
The guide should explain that either `index.ts` needs to be modified by changing the import statement to
`import * as _ from 'lodash';`
Or by changing the Typescript configuration, setting either of the following flags
`"allowSyntheticDefaultImports" : true,`
`"esModuleInterop" : true,`

**Other relevant information:**
webpack version: 4.41.5
typescript version: 3.7.5
lodash version: 4.17.15
@types/lodash: 4.14.149
ts-loader: 6.2.1

### PR DESCRIPTION
**Pull request by EugeneHlushko**
…merge

- mention ts caveat when no default is present in defs (this is ts related but it backfires time to time)
- Closes #3561

### CODE REVIEW

File: src/content/guides/typescript.md [hunk 1]
```diff
@@ -90,6 +91,26 @@ module.exports = {
 
 This will direct webpack to _enter_ through `./index.ts`, _load_ all `.ts` and `.tsx` files through the `ts-loader`, and _output_ a `bundle.js` file in our current directory.
 
+Now let's change the import of `lodash` in our `./index.ts` due to the fact that there is no default export present in `lodash` definitions.
```
Comment (montogeek) [inline src/content/guides/typescript.md]:
```suggestion
Now lets change the import of `lodash` in our `./index.ts` due to the fact that there is no default export present in `lodash` definitions.
```
https://www.grammarly.com/blog/lets-vs-lets/

Comment (wizardofhogwarts) [reply src/content/guides/typescript.md]: This is a great catch

Comment (EugeneHlushko) [reply src/content/guides/typescript.md]: oh man how many time did we get caught on this :D

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**